### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by VikramTiwari**
and fix logo for different device scales

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "angular-resource": "^1.5.2",
     "angular-route": "^1.5.2",
     "angular-sanitize": "^1.5.2",
-    "angular-ui-router": "0.3.1",
+    "angular-ui-router": "^1.0.0-beta.2",
```
Comment (VikramTiwari) [inline package.json]: I did not find any breaking change, but will run it again and if anything is breaking, will downgrade. If not, then will update with fixes for transitions.

File: package.json [hunk 2]
```diff
@@ -43,20 +43,25 @@
...
-    "npm": "^3.x",
+    "npm": "^3.10.7",
+    "protractor": "^4.0.8",
     "q": "latest",
     "request": "latest",
     "shelljs": "latest",
     "snyk": "^1.17.5",
-    "view-helpers": "latest"
+    "view-helpers": "latest",
+    "webpack-dev-server": "^1.15.2"
```
Comment (VikramTiwari) [inline package.json]: Yeah! Not sure how it got here. Will fix it with next push.

File: package.json (hunk 1 above)
Comment (timelf123) [inline package.json]: https://ui-router.github.io/docs/latest/modules/ng1_state_events.html

The only place I found was here, but I also hook state change for our error logging package. Not sure if we add the state change module like the link above, or check ui router for transitions/state like angulartics
```
meanio-circles\public\routes\circles.js:
   19  ])
   20    .run(['$rootScope', '$state', '$http', 'MeanUser', '$cookies', '$location', function($rootScope, $state, $http, MeanUser, $cookies, $location) {
   21:       $rootScope.$on('$stateChangeStart', function(e, toState) {
   22          var acl = MeanUser.acl;
   23          // If the route has a circle requirement on it validate it
```

**Comment by timelf123** [comment]
I've been travelling a lot so no worries! Does anyone else in @linnovate/mean-contributors have thoughts on the $stateChange* events? My thoughts are we add the `ui.router.state.events` for maintainability, but also check for transition vs state events anytime we use them.

Something like:
```
if ($injector.has('$state') && !$injector.has('$transitions')) { }
if ($injector.has('$state') && $injector.has('$transitions')) { }
```

**Comment by timelf123** [comment]
ping @VikramTiwari 
still need to
- [ ] webpack-dev-server -> devdeps
- [ ] angular-ui-router downgrade or patch for transitions (we listen for statechangesuccess a lot)

**Comment by VikramTiwari** [comment]
@timelf123 Could you please confirm and merge.

Also sorry for so much delay, have been pretty busy.

**Pull request merged**